scripts/install

```diff
@@ -259,10 +259,8 @@ ynh_multimedia_addaccess $app
 # Fix app ownerships & permissions
 chown -R $app:www-data "$install_dir"
 chown -R $app: "$data_dir"
-find $install_dir/ -type f -print0 | xargs -r0 chmod 0644
-find $install_dir/ -type d -print0 | xargs -r0 chmod 0755
-find $data_dir/data/ -type f -print0 | xargs -r0 chmod 0640
-find $data_dir/data/ -type d -print0 | xargs -r0 chmod 0750
+chmod -R a=r,u+w,a+X $install_dir/
+chmod -R ug=r,o=---,u+w,ug+X $data_dir/data/
```

Comment:
It feels pretty convoluted to have `ug=r` followed by `u+w` and `ug+X`, we should be able to write directly:

```suggestion
chmod -R u=rwX,g=rX,o=--- $data_dir/data/
```

Reply:
Sadly, it's not equivalent: with your notation, if a file has a x permission for user, this permission x is not removed (yes i know X behavior is very crazy), with mine the x permission is removed cause we first set u=r...

But your comments let's say that adding a comment on that could be a good idea.